assets/css/_swings_view.scss

```diff
@@ -135,3 +135,53 @@
     height: 0.75rem;
   }
 }
+
+@media screen and (max-width: $mobile-max-width) {
+  .m-swings-view__table-header-cell {
+    padding-left: 1rem;
+    padding-right: 1.25rem;
+
+    .subheader {
+      left: 1rem;
+    }
+
+    &.swing-off {
+      padding-left: 1.5rem;
+      padding-right: 1rem;
+
+      .subheader {
+        left: 1.5rem;
+      }
+    }
+
+    .swing-on {
+      display: none;
+    }
+
+    .swing-on-time {
+      position: absolute;
+      top: calc(50% - 7px / 2 + 8px);
```

Comment:
not wild about this use of `px` in here, can we re-express in terms of `rem`?

Reply:
Good call. This and some other values were copied from Figma. I re-did them in `rem` (or `rem` values that are almost exactly equivalent).

Reply:
There was actually a bit of a visual difference so I decided to use the exact, if somewhat clunky, `rem` values.